README.md

```diff
@@ -0,0 +1,295 @@
+> `$ ./avocado-setup.py --run-suite host_sanity,guest_short_sanity --install`
+
+#### Script help output:
+
+```
+$ ./avocado-setup.py -h
+    usage: avocado-setup.py [-h] [--bootstrap] [--run-suite RUN_SUITE]
+                            [--output-dir OUTPUTDIR] [--input-file INPUTFILE]
+                            [--interval-time INTERVAL] [--verbose]
+                            [--only-filter ONLY_FILTER] [--no-filter NO_FILTER]
+                            [--additional-args ADD_ARGS] [--guest-os GUEST_OS]
+                            [--vt {qemu,libvirt}] [--install] [--no-download]
+                            [--no-deps-check] [--install-deps] [--clean]
+                            [--enable-kvm]
+
+    optional arguments:
+      -h, --help            show this help message and exit
+      --bootstrap           Prepare the environment for test
+      --run-suite RUN_SUITE
+                            Indicate which test suite(s) to run
+      --output-dir OUTPUTDIR
+                            Specify the custom test results directory
+      --input-file INPUTFILE
+                            Specify input file for custom mux values for host
+                            tests
+      --interval-time INTERVAL
+                            Specify the interval time between tests
+      --verbose             Enable verbose output on the console
+      --only-filter ONLY_FILTER
+                            Add filters to include specific avocado tests,features
+                            from the guest test suite
+      --no-filter NO_FILTER
+                            Add filters to exclude specific avocado tests,features
+                            from the guest test suite
+      --additional-args ADD_ARGS
+                            Pass additional arguments to the command
+      --guest-os GUEST_OS   Provide Guest os: Default: JeOS.27.ppc64le
+      --vt {qemu,libvirt}   Provide VT: qemu or libvirt Default: libvirt
+      --install             Install the Guest VM, if needed.
+      --no-download         To download the preinstalled guest image
+      --no-deps-check       To force wrapper not to check for dependancy packages
+      --install-deps        To force wrapper to install dependancy packages (Only
+                            for Ubuntu, SLES and yum based OS)
+      --clean               To remove/uninstall autotest, avocado from system
+      --enable-kvm          enable bootstrap kvm tests
+
+```
+
+### Argument Details:
+1. `--bootstrap`:
+
+    > Use this option to bootstrap the environment so that all of the required repositories (avocado, avocado-vt, avocado-misc-tests) are downloaded, installed, and configured.
+    >
+    > Example command to bootstrap the test suite with only host tests.
+    >
+    > `./avocado-setup.py --bootstrap`
+
```

Comment:
Is this needed ?

Reply:
it is good to be in detail, difference between --enable-kvm and default...

Reply:
if you are asking about additional line, yes it does in markdown

Reply:
infact next line is missing it, will add nw...